connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -135,6 +135,8 @@ private void doStart() {
         putIfAbsent(brokerConfig, KafkaConfig.GroupInitialRebalanceDelayMsProp(), 0);
         putIfAbsent(brokerConfig, KafkaConfig.OffsetsTopicReplicationFactorProp(), (short) brokers.length);
         putIfAbsent(brokerConfig, KafkaConfig.AutoCreateTopicsEnableProp(), false);
+        // reduce the size of the log cleaner map to reduce test memory usage
+        putIfAbsent(brokerConfig, KafkaConfig.LogCleanerDedupeBufferSizeProp(), "2097152");
```

Comment:
Makes sense. Just wanted to suggest making it more readable and coincidentally I saw this would make it also same as the one in the equivalent streams class: 

https://github.com/apache/kafka/blob/trunk/streams/src/test/java/org/apache/kafka/streams/integration/utils/EmbeddedKafkaCluster.java#L94

Reply:
Cool, I’ll do the same thing as streams then.

Reply:
Done